pkg/cmd/experimental/syncgroups/groupdetector/groupdetector.go

```diff
@@ -26,6 +26,10 @@ func (l *GroupBasedDetector) Exists(ldapGroupUID string) (bool, error) {
 		return false, err
 	}
 
+	if group == nil {
+		return false, nil
```

Comment:
We haven't set the scope to base object only, that makes sense. We should be, however. I'll look into it.